dojo/settings/settings.dist.py

```diff
@@ -789,6 +789,7 @@ def generate_url(scheme, double_slashes, user, password, host, port, path, param
     'Anchore Engine Scan': ['title', 'severity', 'component_name', 'component_version', 'file_path'],
     'Anchore Grype': ['title', 'severity', 'component_name', 'component_version'],
     'Aqua Scan': ['severity', 'cve', 'component_name', 'component_version'],
+    'AuditJS': ['unique_id_from_tool', 'title', 'description', 'component_version'],
```

Comment:
Unique id from tool must be unique, otherwise it's probably better to use vuln_id_from_tool

Reply:
The ID provided by AuditJS is issued from SonaType OSS Index, which has unique IDs for the vulnerabilities in their database (like 3a3bf289-21dc-4c84-a46e-39280f80bb01)
So they are unique right ?

Reply:
If they wereunique, you wouldn't have to use true other girls in the dedupe Algo. In theory a project can have multiple instances of the same vulnerability, making the id not unique inside the product/application.

Reply:
Alright thanks, I see.
Will change to vuln_id_from_tool then.

Reply:
Done.
Also tested importing the same scan file and the algorithm was able to find duplicates.